CLAUDE.md

```
@@ -0,0 +1,96 @@
+
+## תיאור הפרויקט
+
+**Antigravity Swarm** הוא מערכת multi-agent אוטונומית המבוססת על תזמון משימות מרכזי דרך Supabase, עם agents שפועלים כ-workers על GitHub Actions ו-locally.
+
+המטרה: ביצוע משימות AI מקבילי ועצמאי, עם זיכרון משותף ותיאום בין agents.
+
+---
+
+## ארכיטקטורה
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)